run_tests.sh

```diff
@@ -14,7 +14,7 @@ function echo_task {
 }
 
 REPO_ROOT=`dirname $BASH_SOURCE`
-SERVER_PORT=8003
+SERVER_PORT=8005
```

Comment:
author's note. I changed this to 8005 so it wouldn't interfere with a running edx-platform devstack.